core/misc/tabledrag.js

```diff
@@ -216,8 +216,9 @@ Backdrop.tableDrag.prototype.addColspanClass = function(columnIndex) {
 Backdrop.tableDrag.prototype.hideColumns = function () {
   // Hide weight/parent cells and headers.
   $('.tabledrag-hide', 'table.tabledrag-processed').css('display', 'none');
+  $('table.tabledrag-processed').addClass('tabledrag-handles-shown');
   // Show TableDrag handles.
-  $('.tabledrag-handle', 'table.tabledrag-processed').css('display', '');
```

Comment:
is this change related to the label/machine update? I think it might have slipped in from our other tabledrag issue today.

Reply:
This code attempts to remove the `display: none`, but results in an "empty" `style` attribute in the element (similar to what we resolved in https://github.com/backdrop/backdrop-issues/issues/1849 with empty classes left behind). I can remove it, but does it worth its own issue?

Reply:
...never mind. I've removed this, and filed a PR for it in https://github.com/backdrop/backdrop-issues/issues/5188